test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -1,5 +1,7 @@
 using namespace System.Diagnostics
 
+Import-Module $PSScriptRoot\..\Common\Test.Helpers.psm1
```

Comment:
I suggested higher up because shared tools may be useful for I feel like the directory structure is already a bit too deep and not that intuitive.  At any rate, tools feel like they shouldn't been too hidden.